doc/src/errorrecovery.md

```diff
@@ -0,0 +1,51 @@
+# Error recovery
```

Comment:
Maybe link to here from the quickstart guide?

Reply:
Good idea.